packages/octavius/octavius.0.1.0/opam

```diff
@@ -0,0 +1,16 @@
+opam-version: "1.2"
+maintainer: "leo@lpw25.net"
+authors: "Leo White <leo@lpw25.net>"
+homepage: "https://github.com/ocaml-doc/octavius"
+bug-reports: "https://github.com/ocaml-doc/octavius/issues"
+license: "ISC"
+doc: "http://ocaml-doc.github.io/octavius/"
+tags: ["doc" "ocamldoc" "org:ocaml-doc"]
+dev-repo: "http://github.com/ocaml-doc/octavius.git"
+build: ["ocaml" "pkg/pkg.ml" "build" "--pinned" "%{pinned}%"]
+depends: [
+  "ocamlfind" {build}
+  "ocamlbuild" {build}
+  "topkg" {build & >= "0.7.5"}
+]
+available: [ocaml-version >= "4.01.0"]
```

Comment:
Ok there are actually a couple of other changes that cause problems, so I'll just bump the constraint to `4.03.0`.